src/v/cluster/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ v_cc_library(
     persisted_stm.cc
     tm_stm.cc
```

Comment:
same subject line suggestion

Reply:
already within the desired range `len("cluster/rm_stm: extract a part of rm_stm to be reused from consumer group")==73`